### PR TITLE
5.get api reviews review

### PR DESCRIPTION
**Pull request by Kristaps31**
Please review this code,

It contains one test for valid, and one test for invalid input

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        .expect(200)
+        .then(({body}) => {
+            expect(body.review).toEqual(outcome);
+          })
+    })
+
+    test("400: GET invalid review_id", () => {
+        return request(app)
+        .get("/api/reviews/100")
+        .expect(400)
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: would this be a 400? The review id 100 could exist, but doesnt yet.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        .then(({body})=> {
+            expect(body.msg).toBe("Bad Request");
+        });
+    })
```
Comment (Beth-McCormack) [inline __tests__/app.test.js]: can you add a test for invalid id eg api/reviews/banana. This is different from above as the review id of bananas should never exist.

**Review by Beth-McCormack — CHANGES_REQUESTED** [review]
see comment

**Review by yalicee — APPROVED** [review]
Great code!!! Keep it up

**Pull request merged**